pages/templates/partials/initiative_snippet_result.html

```diff
@@ -13,7 +13,7 @@
 
           <div class="card__content full initiative">
             <div>
-              <span class="card__item name bold tight"><span>{{ initiative.title }}: {{ initiative.name }}</span></span>
+              <span class="card__item name bold tight"><span>{{ initiative.vote|yesno|title }} on {{ initiative.title }}: {{ initiative.name }}</span></span>
```

Comment:
ditto, use new property?

Reply:
These don't all have pages attached and it's a bit confusing what will have what so I just left as-is here because it already supports legacy. I just added vote to be more consistent across pages.